### DISCUSSION TITLE
Warnings in FF81

### DISCUSSION THREAD

**Discussion by GeoBau** [chartjs/chartjs-plugin-annotation#770] [Q&A]
I get a warning:
FF: "Farbe erwartet, aber 'undefined' gefunden."
(translated "Color expected, but 'undefined' found")
Yon can test it: samples>Line Annotation>Basic
The warning has something to do with lines..

A similiar thing happens in labels with font: { weight:"light" },
FF: Fehler beim Verarbeiten des Wertes für 'font'.  Deklaration ignoriert.
(translated: "Error for the value of 'font'. declaration ignored.")
You can test it: samples>label  annotations>point (add in font weight:"light")
Nevertheless weight is working.

These warnings come only in FF not in Chrome.
Any idea how to get rid of the messages?

**Comment by LeeLenaleee** [comment]
FireFox 81 is almost 2 years old. If you update to the most recent version there are no errors. So to get rid of them you will need to update your browser.

**Comment by GeoBau** [comment]
thanks  - the administrator deactivated the automatic update. I will ask him for update!

**Comment by GeoBau** [comment]
I have now Update to FF102.
samples>Line Annotation>Basic
Warning comes up:  "Expected color but found ‘undefined’."

In my case I have another warning (it only happens with annotations)
Error in parsing value for ‘font’.  Declaration dropped. 

I am using cross hair. In my case the warnings appear whenever I move the cross hair.
Whereas in the examples the message come up only once at first load.

**Reply by LeeLenaleee** [reply]
When I open the specified sample in FF or Chrome it does not give me those errors so without any reproducable sample I can't help you any further

**Reply by GeoBau** [reply]
![annotation](https://user-images.githubusercontent.com/109003222/180781890-b86608e7-d6a6-4cbd-88ff-8010a18b717b.JPG)
I made a screenshot maybe that helps...
I am using FF 102.0.1. (64-bit)
Thanks!

**Reply by LeeLenaleee** [reply]
No it does not, only thing I can think off is that somehow maby you have installed some plugin in your browser that conflicts and messes something up, since I cannot reproduce this with a clean FF 102.0.1 (64 bit)

**Reply by GeoBau** [reply]
I just noticed that there is an error on top ... maybe this gives an idea.
The warning appears by moving the cursor to a new data point (then the warning appears 30-50x) .
I switched off all extensions in FF, but that does not make an improvement.
But it is just a warning ... :-)
![image](https://user-images.githubusercontent.com/109003222/180787188-78c2b4ce-e3d6-415f-b934-41e076515517.png)

**Reply by LeeLenaleee** [reply]
You will just have to ignore it, switch to another browser or supply a reproducable sample since I can't reproduce it no matter what I do with my mouse on the official sample

**Reply by GeoBau** [reply]
Thanks for your efforts... Ciao
